lib/validator.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+class Validator {
+  isString(input) {
```

Comment:
Between line 3 and 4 their should be a part called `constructor` which is a function use within class to help mange 'this'